tfhe/src/integer/gpu/mod.rs

```diff
@@ -273,6 +281,164 @@ pub unsafe fn unchecked_scalar_mul_integer_radix_kb_async<T: UnsignedInteger, B:
     );
 }
 
+#[allow(clippy::too_many_arguments)]
+/// # Safety
+///
+/// - [CudaStreams::synchronize] __must__ be called after this function as soon as synchronization
+///   is required
+pub unsafe fn compression_compress_integer_radix_async<T: UnsignedInteger>(
```

Comment:
Maybe we could drop the `compression_` prefix?

Reply:
https://github.com/zama-ai/tfhe-rs/pull/1424#discussion_r1711563330